solutions/functions/min-max-range.js

```diff
@@ -0,0 +1,76 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+
+// ## This function should return single integer. The integer should be maximum value of input list
+// ## Please fill the following function
+function maximumValue(input_numbers){
+    // write below this here
+    let maxValue = Number.NEGATIVE_INFINITY;
+    for(let i = 0; i < input_numbers.length; i++){
+        if(input_numbers[i] > maxValue) maxValue = input_numbers[i];
```

Comment:
@Abduliitr Please use curly braces. Same for the other function.

Reply:
Added.